zetta_utils/mazepa_layer_processing/common/subchunkable_apply_flow.py

```diff
@@ -210,6 +212,12 @@ def build_subchunkable_apply_flow(  # pylint: disable=keyword-arg-before-vararg,
             )
         bbox_ = bbox
 
+    if dst is None:
+        if not skip_intermediaries:
+            raise ValueError("skip_intermediaries must be True when dst is None.")
```

Comment:
"`skip_intermediaries` must be True when `dst` is None." would be better here. (Just backticks for variable name.)